api/tests/integration/helpers.py

```diff
@@ -49,3 +50,36 @@ def create_feature_with_api(
         content_type="application/json",
     )
     return create_standard_feature_response.json()["id"]
+
+
+def get_env_feature_states_list_with_api(client: APIClient, query_params: dict) -> dict:
+    """
+    Returns feature states using the provided test client.
+
+    :param client: DRF api client to use to make the request
+    :param query_params: A Mapping object used as query params for filtering
+
+    """
+    url = reverse(
+        "api-v1:features:featurestates-list",
```

Comment:
Really minor now but it looks like this has unnecessarily wrapped? You could cut this function from 4 lines to 2 :)